Update table_data.rst with "sequence" and "exclude" examples (#765)
jieter/django-tables2: docs/pages/table-data.rst

When using this approach, the following options might be useful to customize
what fields to show or hide:

> - `~.Table.Meta.sequence` -- reorder columns (if used alone, columns that are not specified are still going to be shown in the table after the specified columns)

Reviewer: ```suggestion
- `~.Table.Meta.sequence` -- reorder columns (if used alone, columns that are not specified are still rendered in the table after the specified columns)
```

Author: Thanks, agreed with your suggestion.

- `~.Table.Meta.fields` -- specify model fields to *include*
- `~.Table.Meta.exclude` -- specify model fields to *exclude*

In this example we will demonstrate how this can be done::
    
    # tables.py

>     class PersonTable(tables.Table):
>     class Meta:
>         model = Person
>         sequence = ("last_name", "first_name", "birth_date", )
>         exclude = ("user", )

Reviewer: Besides the indentation issue for which I suggest a change below, I think we need to describe what the user should expect from this example. Just adding an example does not clarify the interaction of `sequence`, `fields` and `exclude` much.

Can you add such an explanation?

```suggestion
    class PersonTable(tables.Table):
        class Meta:
            model = Person
            sequence = ("last_name", "first_name", "birth_date", )
            exclude = ("user", )
```

Author: Oops, thanks for catching this. I've accepted the suggestion and will now add an explanation.

Author: @jieter Explanation added - let me know if that's what you had in mind. I'm happy to revise it further if needed.
